package.json

```diff
@@ -5,7 +5,7 @@
     "postinstall": "husky install && patch-package && nx run-many --target=install:isolated",
     "reset:install": "git checkout origin/master package-lock.json && npm ci",
     "clean:install": "rimraf **/node_modules && npm run reset:install",
-    "dev:atomic": "concurrently \"nx run headless:dev\" \"nx run atomic:dev\"",
+    "dev:atomic": "concurrently \"npm run -w @coveo/headless dev\" \"npm run -w @coveo/atomic dev\"",
```

Comment:
Without this change, this would run Nx twice in parallel, which could cause conflicts.

Nx has some mechanisms to handle dev tasks, but they are more complex. IMO, it wouldn't be a valuable use of our time.

Reply:
Can't `nx run-many --target=dev` be used with selected workspaces/projects?